caching/src/main/java/com/iluwatar/caching/AppManager.java

```diff
@@ -50,6 +50,8 @@ public class AppManager {
    * Cache Store.
    */
   private final CacheStore cacheStore;
+  private final CacheStoreRead cacheStoreRead;
+  private final CacheStoreWrite cacheStoreWrite;
```

Comment:
It's a good inititiative to refactor this, but I find it confusing that we now have three different cache classes and objects.

Reply:
One option would be to create a cache class for each of the use cases we want to demonstrate (write-through, write-around, write-behind, cache-aside, read-through)